sunpy/map/tests/test_compositemap.py

```diff
@@ -65,6 +65,27 @@ def test_plot_composite_map_colors(composite_test_map):
 
 
 def test_plot_composite_map_mplkwargs(composite_test_map):
```

Comment:
What is this testing?

Reply:
It is testing if the supported mpl contour kwargs which are passed through plot_settings are properly reflected or not

Reply:
This is just checking if dictionary assignment works tho?

Reply:
Yes can simply say that

Reply:
Shall I change the test name?

Reply:
So unless it's testing something related to the code changes in this PR, I don't see the benefit of this test.

Reply:
So it's essentially not testing anything to do with the code changes in this PR?

Reply:
Yes, the test needs to be changed to be meaningful. What it should do, I don't know

Reply:
I have modified the tests, please review them.

Reply:
I would also check that the work arounds for cmap etc are correct for normal maps instead of just composite maps.

We have to make sure these do not break any existing map calls for draw_contours

Reply:
I included `contour_args = {'norm': 'log'}` to confirm whether the original `contour_args` from `draw_contours` take precedence over `plot_settings`. Should I do the same for `cmap`, or would it be better to create a new test specifically for this?

Reply:
Do the same for cmap in the same test.

Reply:
Added.